### PR TITLE
QOL improvements: add helper function to load arbitrary img, update `notes` flag in training script

### PR DESCRIPTION
**Pull request by robertklee**
This is a very small PR that adds some additional functionality to the eval code to load arbitrary images.

### CODE REVIEW

**Review by nicolepeverley — APPROVED** [review]
LGTM 👍

**Comment by robertklee** [comment]
lol i forgot to push the important actual commit 😂 my bad

File: evaluation.py [hunk 1]
```diff
@@ -35,7 +37,18 @@ def predict_heatmaps(self, h, X):
...
+        predict_mean = np.mean(predict_heatmaps)
+        predict_max = np.max(predict_heatmaps)
+        predict_min = np.min(predict_heatmaps)
+        predict_var = np.var(predict_heatmaps.flatten())
+        print('Mean: {:0.6e}\t Max: {:e}\t Min: {:e}\t Variance: {:e}'.format(predict_mean, predict_max, predict_min, predict_var))
+        normalized_heatmaps = predict_heatmaps / predict_max
+        normalized_heatmaps = normalized_heatmaps - predict_min
+        normalized_heatmaps = predict_heatmaps / np.max(predict_heatmaps)
+
+        return normalized_heatmaps
```
Comment (nicolepeverley) [inline evaluation.py:51]: Do the normalized predicted heatmaps look different or is this just to normalize the np array?

**Review by nicolepeverley — COMMENTED** [review]
lol I thought you were just updating the comments haha, still looks good just had a  question!

File: evaluation.py (hunk 1 above)
Comment (robertklee) [reply evaluation.py:51]: this is to normalize the array. We're getting some major issues with large numbers in the network right now so this is to try to combat that in order to plot it.

**Comment by robertklee** [comment]
haha i had soft reset it to amend my commit, but forgot to force push afterwards

File: evaluation.py (hunk 1 above)
Comment (robertklee) [reply evaluation.py:51]: do you remember the really odd looking heat map? So normally our network output is ~0 to ~1, but occasionally it outputs like 6.0e16 and -6.0e16 (which results in that really blurry image we saw)

Comment (nicolepeverley) [reply evaluation.py:51]: oh right sweet so this should keep it between the normal output

Comment (robertklee) [reply evaluation.py:51]: yep exactly. I have to fix our model too so we dont even get to this state

**Comment by robertklee** [comment]
gonna merge since it's a small change. lmk if i missed anything and I'll add it to my next PR!

File: evaluation.py [hunk 2]
```diff
@@ -35,7 +37,18 @@ def predict_heatmaps(self, h, X):
...
-        return predict_heatmaps
+
+        print('model prediction metrics: ')
+        predict_mean = np.mean(predict_heatmaps)
+        predict_max = np.max(predict_heatmaps)
+        predict_min = np.min(predict_heatmaps)
+        predict_var = np.var(predict_heatmaps.flatten())
+        print('Mean: {:0.6e}\t Max: {:e}\t Min: {:e}\t Variance: {:e}'.format(predict_mean, predict_max, predict_min, predict_var))
+        normalized_heatmaps = predict_heatmaps / predict_max
+        normalized_heatmaps = normalized_heatmaps - predict_min
```
Comment (julianrocha) [inline evaluation.py:48]: These two lines get overwritten by the next line

**Pull request merged**